Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.cpp

```diff
@@ -75,25 +76,13 @@ namespace AzToolsFramework
         PrefabSaveHandler::PrefabSaveHandler()
         {
             s_prefabLoaderInterface = AZ::Interface<PrefabLoaderInterface>::Get();
-            if (s_prefabLoaderInterface == nullptr)
-            {
-                AZ_Assert(false, "Prefab - could not get PrefabLoaderInterface on PrefabSaveHandler construction.");
-                return;
```

Comment:
Any particular reason for removing these returns? Early returns will prevent constructing an invalid object with nullptrs to interfaces.

Reply:
Thanks for point this out. I looked into [PrefabFocusHandler.cpp#L62](https://github.com/aws-lumberyard-dev/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp#L62) and was wondering why the code is not consistent. I will revert the changes.